gateway/handler_codec.go

```diff
@@ -119,7 +118,7 @@ func (i *handler) serveCodec(ctx context.Context, w http.ResponseWriter, r *http
 	if !ok {
 		// This is never supposed to happen unless function is called with wrong parameters.
 		err := fmt.Errorf("unsupported content type: %s", requestedContentType)
```

Comment:
(since this is user-input, i think?)
```suggestion
		err := fmt.Errorf("unsupported content type: %q", requestedContentType)
```